api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/Responses.scala

```diff
@@ -35,75 +39,59 @@ object ResultResponse {
 
 @Schema(
   name = "ResultList",
-  description = "A paginated list of works."
+  description = "A paginated list of documents."
```

Comment:
Documents is very Elastic...
```suggestion
  description = "A paginated list of results."
```
❓